QuizApp/Core/Custom UI/GPADetailViewController/GPADetailViewController.swift

```diff
@@ -0,0 +1,166 @@
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.register(ScoreCell.self, forCellReuseIdentifier: ScoreCell.reuseIdentifier)
+        view.addSubview(tableView)
+    }
+    
+    // MARK: - Constraints Setup
+    
+    private func setupTableViewConstraints() {
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: pointLabel.bottomAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: Constants.TableView.bottomAnchor)
+        ])
+    }
+    
+    private func setupPointLabelConstraints() {
+        NSLayoutConstraint.activate([
+            pointLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: Constants.PointLabel.topAnchor),
+            pointLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Constants.PointLabel.leadingAnchor),
+        ])
+    }
+    
+    private func setupStarImageConstraints() {
+        NSLayoutConstraint.activate([
+            starImage.topAnchor.constraint(equalTo: view.topAnchor, constant: Constants.StarImage.topAnchor),
+            starImage.leadingAnchor.constraint(equalTo: pointLabel.trailingAnchor, constant: Constants.StarImage.leadingAnchor),
+        ])
+    }
+    
+    private func noPointsImageConstraints() {
+        NSLayoutConstraint.activate([
+            noPointsImage.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            noPointsImage.centerXAnchor.constraint(equalTo: view.centerXAnchor)
+        ])
+    }
+}
+
+// MARK: - UITableViewDataSource
+
+extension GPADetailViewController: UITableViewDataSource {
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        subjects.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: ScoreCell.reuseIdentifier, for: indexPath) as! ScoreCell
+        let subject = subjects[indexPath.row]
+        cell.configure(with: subject)
+        return cell
+    }
+}
+
+// MARK: - UITableViewDelegate
+
+extension GPADetailViewController: UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        let cellHeight = Constants.TableView.cellHeight
+        let distanceBetweenCells = Constants.TableView.distanceBetweenCells
+        let totalHeight = cellHeight + distanceBetweenCells
+        return totalHeight
+    }
+}
+
+// MARK: - Constants
+
+private extension GPADetailViewController {
+    enum Constants {
+        
+        enum PointLabel {
+            static let textColor: UIColor  = .black
+            static let text = "დაგროვილი ქულები"
+            static let font: UIFont = .systemFont(ofSize: 16)
```

Comment:
აღარ დაგიკომენტარებ ყველგან ამას, redundant ცვლადის ტიპები მოაშორე, არაა საჭირო
```suggestion
            static let font = .systemFont(ofSize: 16)
```